testproject/Assets/Tests/Runtime/MultiprocessRuntime/BaseMultiprocessTests.cs

```diff
@@ -135,17 +163,45 @@ public virtual void Teardown()
         [OneTimeTearDown]
         public virtual void TeardownSuite()
         {
+            MultiProcessLog($"TeardownSuite");
             if (!IgnoreMultiprocessTests)
             {
+                MultiProcessLog($"TeardownSuite - ShutdownAllProcesses");
                 MultiprocessOrchestration.ShutdownAllProcesses();
+                MultiProcessLog($"TeardownSuite - NetworkManager.Singleton.Shutdown");
                 NetworkManager.Singleton.Shutdown();
                 Object.Destroy(NetworkManager.Singleton.gameObject); // making sure we clear everything before reloading our scene
+                MultiProcessLog($"Currently active scene {SceneManager.GetActiveScene().name}");
+                MultiProcessLog($"m_OriginalActiveScene.IsValid {m_OriginalActiveScene.IsValid()}");
                 if (m_OriginalActiveScene.IsValid())
                 {
                     SceneManager.SetActiveScene(m_OriginalActiveScene);
                 }
+                MultiProcessLog($"TeardownSuite - Unload {BuildMultiprocessTestPlayer.MainSceneName}");
                 SceneManager.UnloadSceneAsync(BuildMultiprocessTestPlayer.MainSceneName);
+                MultiProcessLog($"TeardownSuite - Unload {BuildMultiprocessTestPlayer.MainSceneName}");
+            }
+        }
+
+        public static void MultiProcessLog(string msg)
```

Comment:
nit: https://docs.unity3d.com/ScriptReference/Logger.html

Reply:
I'm not clear on how to use this to control the logging format just yet.